src/components/Menu/VersionSwitcher/index.tsx

```diff
@@ -48,10 +48,20 @@ export function LibVersionSwitcher({
   alternativeVersion: string;
   primaryVersion: string;
 }) {
+  console.log(url, alternativeVersion, primaryVersion);
```

Comment:
Is leaving this console.log in intentional?

Reply:
Nope. My bad. Got it cleaned up